### ISSUE TITLE
Unable to use version 3 with Next.js

### ISSUE THREAD

**Issue by chetbox** [sindresorhus/osx-release#10]
Hi! Thanks for this helpful module.

We have a site uses this library to give user information about machines associated to their account on a web dashboard built in Next. We use a pretty basic Next.js 11 setup to create a static site with `next export`. 

To support Monterey and to get the benefit of #9 we've upgraded to 3.0.1 but I'm struggling to get Next.js to work with it.

This is the error I get:

```
error - node:os
Module build failed: UnhandledSchemeError: Reading from "node:os" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
```

I've also followed your guide on using ESM modules but Next.js does not seem happy with that at all, even using the [ESM externals feature](https://github.com/vercel/next.js/pull/27069).

How do I use this module client side with Next.js?

**Comment by sindresorhus** [comment]
Next.js 12 supports ESM. See their announcement.

**Comment by chetbox** [comment]
I believe Next 12 has simply enabled the ESM externals feature by default.

With an empty Next.js 12.0.3 Typescript project with a dependency on `macos-release` 3.0.1 I get the same error:

```
node:os
Module build failed: UnhandledSchemeError: Reading from "node:os" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
```

This is my component:

```tsx
import type { NextPage } from 'next'
import styles from '../styles/Home.module.css'
import macosRelease from 'macos-release'

const Home: NextPage = () => {
  return (
    <div className={styles.container}>
        {macosRelease('13.2.0')?.name}
    </div>
  )
}

export default Home
```

**Comment by chetbox** [comment]
This is still proving to be a real headache. I think it's to do with with webpack treating `node:os` as a URL import but I'm not sure. (Next.js 11.1.2 uses `@svgr/webpack` 5.5.0 and `node:` import support was added to `webpack` [in 5.40.0](https://github.com/webpack/webpack/pull/12693) I believe.)

For now my workaround is to use patch `index.js` to import `os` and move on. I've added `replace-in-file` as a dev dependency and added a script to `package.json` that looks like this:

```json
  "scripts": {
    "postinstall": "replace-in-file \"import os from 'node:os'\" \"import os from 'os'\" node_modules/macos-release/index.js",
    ...
}
 ```
